aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/implementing-basic-crud-functionality-with-the-entity-framework-in-asp-net-mvc-application.md

```diff
@@ -159,7 +159,7 @@ In the following code, `courseID` doesn't match a parameter in the default route
 
    The new code reads the existing entity and calls <xref:System.Web.Mvc.Controller.TryUpdateModel%2A> to update fields from user input in the posted form data. The Entity Framework's automatic change tracking sets the [EntityState.Modified](<xref:System.Data.EntityState.Modified>) flag on the entity. When the [SaveChanges](https://msdn.microsoft.com/library/system.data.entity.dbcontext.savechanges(v=VS.103).aspx) method is called, the <xref:System.Data.EntityState.Modified> flag causes the Entity Framework to create SQL statements to update the database row. [Concurrency conflicts](handling-concurrency-with-the-entity-framework-in-an-asp-net-mvc-application.md) are ignored, and all columns of the database row are updated, including those that the user didn't change. (A later tutorial shows how to handle concurrency conflicts, and if you only want individual fields to be updated in the database, you can set the entity to [EntityState.Unchanged](<xref:System.Data.EntityState.Unchanged>) and set individual fields to [EntityState.Modified](<xref:System.Data.EntityState.Modified>).)
 
-   To prevent overposting, the fields that you want to be updateable by the Edit page are whitelisted in the `TryUpdateModel` parameters. Currently there are no extra fields that you're protecting, but listing the fields that you want the model binder to bind ensures that if you add fields to the data model in the future, they're automatically protected until you explicitly add them here.
+   To prevent overposting, the fields that you want to be updateable by the Edit page are allowlisted in the `TryUpdateModel` parameters. Currently there are no extra fields that you're protecting, but listing the fields that you want the model binder to bind ensures that if you add fields to the data model in the future, they're automatically protected until you explicitly add them here.
```

Comment:
```suggestion
   To prevent overposting, the fields that you want to be updateable by the Edit page are listed in the `TryUpdateModel` parameters. Currently there are no extra fields that you're protecting, but listing the fields that you want the model binder to bind ensures that if you add fields to the data model in the future, they're automatically protected until you explicitly add them here.
```